### PR TITLE
Fix the README for SGEMM example

### PR DESCRIPTION
**Pull request by yuyu2172**
The current README has some ambiguity in describing the purpose of the kernel.
I updated the README to be clear that the kernel needed to configure `grid size` and `block size` directly.

### CODE REVIEW

File: examples/gemm/README.md [hunk 1]
```diff
@@ -15,21 +15,23 @@ python sgemm.py [--gpu GPU_ID] [--m m] [--n n] [--k k]
...
+The SGEMM kernel expects input and output arrays to be in Fortran contiguous memory layout, and this layout is enforced by `cupy.asfortranarray`.
 
 For compilation, `load_kernel` is used to compile a CUDA code written in `sgemm.cu`.
 This function takes a text of code and name of the kernel as input and returns `cupy.cuda.Function` object.
 The compiled code is cached, and it avoids the compilation process after the first time.
 Also, the CUDA code can be modified at Python level because it is simply a text.
-In this example, C macros that determines distribution of data to threads are specified at runtime.
+In this example, C macros that determine distribution of data to threads are specified at runtime.
 
 Back to `cupy.cuda.Function`, this object allows you to call the kernel with CUDA's `cuLaunchKernel` interface.
```
Comment (takagi) [inline examples/gemm/README.md]: Would it be good to place a heading to show the point of the paragraph?

```
#### How to supply grid size, block size and shared memory size on launching a kernel function
```

File: examples/gemm/README.md [hunk 2]
```diff
@@ -15,21 +15,23 @@ python sgemm.py [--gpu GPU_ID] [--m m] [--n n] [--k k]
...
 ### What this demo contains
 
-CuPy arrays can be used without abstractions such as `ElementwiseKernel` if convenient `ndarray` interface is not necessary.
-In this example, we work at the level of linear arrays (`float*`) and `cuLaunchKernel` to call a SGEMM kernel that exploits the predefined memory layout.
+In this example, we work on a SGEMM kernel that requires a complete interface to `cuLaunchKernel` (e.g. grid size and size of shared memory), which is not provided by `cupy.ElementwiseKernel`.
+CuPy arrays work regardless of the underlying memory layouts thanks to `ndarray` abstraction.
+As is the case for this example, `ndarray` abstraction does not need to be used if the underlying memory layouts of arrays match the ones expected by a kernel.
+The SGEMM kernel expects input and output arrays to be in Fortran contiguous memory layout, and this layout is enforced by `cupy.asfortranarray`.
 
 For compilation, `load_kernel` is used to compile a CUDA code written in `sgemm.cu`.
```
Comment (takagi) [inline examples/gemm/README.md]: Would it be good to place a heading to show what readers would want to know?
```
#### How to dynamically compile and launch a kernel function written in CUDA C
```

File: examples/gemm/README.md [hunk 3]
```diff
@@ -15,21 +15,23 @@ python sgemm.py [--gpu GPU_ID] [--m m] [--n n] [--k k]
...
 This function takes a text of code and name of the kernel as input and returns `cupy.cuda.Function` object.
 The compiled code is cached, and it avoids the compilation process after the first time.
 Also, the CUDA code can be modified at Python level because it is simply a text.
-In this example, C macros that determines distribution of data to threads are specified at runtime.
+In this example, C macros that determine distribution of data to threads are specified at runtime.
 
 Back to `cupy.cuda.Function`, this object allows you to call the kernel with CUDA's `cuLaunchKernel` interface.
 In other words, you have control over grid size, block size, shared memory size and stream id.
-At this level of interface, it becomes straightforward to transfer `.cpp` code that calls CUDA kernels to CuPy code.
+At this level of interface, it becomes straightforward to transfer host `.cu` that calls CUDA kernels to Python.
```
Comment (takagi) [inline examples/gemm/README.md]: "replace" may sound more natural than "transfer" here.

**Comment by yuyu2172** [comment]
Thank you very much for your comments!
I updated the file.

**Comment by takagi** [comment]
LGTM! I will merge this PR after CuPy v2.0.0a1 release because the master branch is frozen for it.

**Comment by takagi** [comment]
Backporting is not needed because sgemm example is introduced at v2.0.0a1.
